src/core/api/fims/admin/_swagger.json.tpl

```diff
@@ -0,0 +1,463 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenID Provider",
+    "description": "OpenID Provider service",
+    "version": "0.0.1"
+  },
+  "host": "api.openid-provider.io.italia.it",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/admin/clients": {
```

Comment:
@Krusty93 how is exposed the path on api management?
I suggest to add a fims prefix because api management does not support duplicate base paths and `/admin/clients` is too generic and we'll locked for future implementatiobs